Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/SphereComponentMode.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/ComponentModes/SphereComponentMode.h>
+
+#include <AzToolsFramework/ComponentModes/ShapeTranslationOffsetViewportEdit.h>
+#include <AzToolsFramework/Manipulators/RadiusManipulatorRequestBus.h>
+#include <AzToolsFramework/Manipulators/ShapeManipulatorRequestBus.h>
+#include <AzToolsFramework/ViewportSelection/EditorSelectionUtil.h>
+
+namespace AzToolsFramework
+{
+    void InstallSphereViewportEditFunctions(SphereViewportEdit* sphereViewportEdit, const AZ::EntityComponentIdPair& entityComponentIdPair)
+    {
+        auto getSphereRadius = [entityComponentIdPair]()
+        {
+            float radius = 0.5f;
```

Comment:
Default sphere radius?

Reply:
That's a good idea. I've already started an AR run so if that succeeds I'll submit this and make that change in the next PR.

Reply:
AR run failed for something unrelated, so I'll update in this change

Reply:
Looking over the various definitions of shapes throughout the engine, there are defaults and hardcoded values all over the place (although as far as I can tell they always use the same values, just not shared constants). I'm going to do a separate PR to tidy that up.